tests/model/test_model.py

```diff
@@ -1566,6 +1566,23 @@ def test__handle_subscription_event_subscribers(self, model, mocker,
         new_subscribers = model.stream_dict[event['stream_id']]['subscribers']
         assert new_subscribers == expected_subscribers
 
+    @pytest.mark.parametrize('event', [
+        ({'type': 'subscription', 'op': 'peer_add',
+          'stream_id': 462, 'user_id': 12}),
+        ({'type': 'subscription', 'op': 'peer_remove',
+          'stream_id': 462, 'user_id': 12}),
+    ], ids=[
+        'peer_subscribed_to_stream_that_user_is_unsubscribed_to',
+        'peer_unsubscribed_from_stream_that_user_is_unsubscribed_to',
+    ])
+    def test__handle_subscription_event_subscribers_to_unsubscribed_streams(
+                            self, model, mocker, stream_dict, event):
+        model.stream_dict = stream_dict
+
+        model._handle_subscription_event(event)
+
+        # No exceptions should be raised - tested without any assertion.
+
```

Comment:
I'm not sure if there's a correct way to test this. The missing check previously raised a `KeyError` because the stream id was missing in `stream_dict`. So I need to test if a `KeyError` is not raised, which can be tested without an assertion?

Regardless, the call to `_handle_subscription_event` does not do anything at all so there isn't anything to be asserted in the first place.